composer.json

```diff
@@ -77,7 +77,7 @@
         "drupal/field_permissions": "^1.2",
         "lesstif/php-jira-rest-client": "2.6.0",
         "drupal/jira_rest": "^4.0@beta",
-        "drupal/ckeditor5_embedded_content": "^1.0@alpha",
+        "drupal/ckeditor5_embedded_content": "1.0.0-alpha3",
```

Comment:
ck5 embed has a custom local patch for all SDP projects, that is why this version needs to be locked so that the patch gets applied